src/Core/src/ImageSources/iOS/ImageSourceExtensions.cs

```diff
@@ -146,10 +147,32 @@ internal static UIImage GetPlatformImage(this CGImageSource cgImageSource, int s
 				if (cgimage is null)
 					throw new InvalidOperationException("Unable to create CGImage from CGImageSource.");
 
-				image = new UIImage(cgimage, scale, UIImageOrientation.Up);
+				image = new UIImage(cgimage, scale, ToUIImageOrientation (cgImageSource));
 			}
 
 			return image;
 		}
+
+		static UIImageOrientation ToUIImageOrientation (CGImageSource cgImageSource)
```

Comment:
The code format is not quite right :) seems mono days? we provide the rules in a .editorconfig.. 

Our dotnet format on this file only will also work.